src/domains/mil-app-poc/api/emd_mil_testing/openapi.mil.tpp.yml

```
@@ -0,0 +1,342 @@
+openapi: 3.0.3
+
+info:
+  title: EMD TPP NETWORK TESTING API
+  version: '1.0'
+  description: |-
+    EMD TPP NETWORK TESTING
+  contact:
+    name: PagoPA S.p.A.
+    email: cstar@pagopa.it
+
+servers:
+  - description: Development Test
```

Each apply should expose only one server:
```suggestion
  - url: https://${hostname}
    description: Generated server url
```